src/scripts/App.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const App = () => <div className="App">This is the App</div>;
```

Comment:
It's a common practice to use .jsx format for files with React and should be named with uppercased first letter (main.js)